include/hipSYCL/sycl/handler.hpp

```diff
@@ -1182,6 +1191,10 @@ class handler {
         has_non_instant_dependency || is_unbound ||
         !is_dedicated_in_order_queue ||
         op->is_requirement()) {
+      if (ACPP_FORCE_INSTANT_SUBMISSION) {
```

Comment:
Why not `#if` ?

Reply:
Same style as in line 1190, and this way the conditional code is still syntax / type checked when disabled - but no strong preference from my side.

Reply:
I think in 1190 we do this because there are a lot of other runtime conditions, which makes it a bit more cumbersome to use preprocessor `#if`. I think in this case, we should perhaps better use `#if`.